### PR TITLE
CI: Switch from Travis to Github Actions

### PR DESCRIPTION
**Pull request by rbuch**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
```
Comment (matthiasdiener) [inline .github/workflows/test.yml:14]: Could use a matrix here to test various java versions (and maybe OSs)

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      - name: Build with Gradle
+        run: gradle build
+      - name: Setup xvfb
+        run: sudo apt-get install xvfb
```
Comment (evan-charmworks) [inline .github/workflows/test.yml:18]:
```suggestion
        run: sudo apt-get update && sudo apt-get install xvfb
```

File: .github/workflows/test.yml (hunk 1 above)
Comment (rbuch) [reply .github/workflows/test.yml]: Added.

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+name: Projections CI
+
+on: [pull_request]
```
Comment (matthiasdiener) [inline .github/workflows/test.yml:3]:
```suggestion
on: pull_request
```

**Review by matthiasdiener — APPROVED** [review]

**Review by matthiasdiener — DISMISSED** [review]

**Review by matthiasdiener — APPROVED** [review]

**Review by evan-charmworks — APPROVED** [review]

**Pull request merged**